packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -33,30 +33,52 @@
 // postcss-bem-linter: define text-field
 
 .mdc-text-field {
+  @include mdc-ripple-surface;
+  // Text Field intentionally omits press ripple, so each state needs to be specified individually.
+  @include mdc-states-base-color($mdc-text-field-ink-color);
+  @include mdc-states-hover-opacity(mdc-states-opacity($mdc-text-field-ink-color, hover));
+  @include mdc-states-focus-opacity(mdc-states-opacity($mdc-text-field-ink-color, focus));
+  @include mdc-ripple-radius-bounded;
+
+  // Shape
+  @include mdc-text-field-corner-radius($mdc-text-field-border-radius);
+
+  // Colors
+  @include mdc-text-field-label-color($mdc-text-field-label);
+  @include mdc-text-field-ink-color($mdc-text-field-ink-color);
+  @include mdc-text-field-caret-color(primary);
   @include mdc-text-field-bottom-line-color($mdc-text-field-bottom-line-idle);
   @include mdc-text-field-hover-bottom-line-color($mdc-text-field-bottom-line-hover);
   @include mdc-text-field-line-ripple-color_(primary);
-  @include mdc-text-field-ink-color($mdc-text-field-ink-color);
-  @include mdc-text-field-label-color($mdc-text-field-label);
-  @include mdc-text-field-helper-text-color($mdc-text-field-helper-text-color);
   @include mdc-text-field-fullwidth-bottom-line-color($mdc-text-field-fullwidth-bottom-line-color);
+  @include mdc-text-field-helper-text-color($mdc-text-field-helper-text-color);
   @include mdc-text-field-icon-color($mdc-text-field-icon-color);
-  @include mdc-text-field-caret-color(primary);
+  @include mdc-text-field-fill-color($mdc-text-field-background);
+
+  // Floating Label
+  @include mdc-text-field-floating-label_;
+  // The following mixins will be removed after select is updated to default to the box.
```

Comment:
I don't understand this comment or the shake-animation/keyframes taking `text-field-box` as an identifier. The mdc-select package doesn't depend on mdc-text-field at all, so what is this doing? The only reference I see to this within Select's own styles is with `text-field-outlined`.

Reply:
This comment is stating that this isn't being updated until the floating-label is updated after the select. Updating these requires me to change things in the floating-label package, which is not within the scope of this PR.